docs/querying/sql-functions.md

```diff
@@ -1309,11 +1373,33 @@ Returns the relative rank of the row calculated as a percentage according to the
 
 ## POSITION
 
-`POSITION(<CHARACTER> IN <CHARACTER> [FROM <INTEGER>])`
+Returns the one-based index position of a substring within an expression, optionally starting from a given one-based index. If `substring` is not found, returns 0.
 
-**Function type:** [Scalar, string](sql-scalar.md#string-functions)
+* **Syntax**: `POSITION(substring IN expr [FROM startingIndex])`
+* **Function type:** Scalar, string
+
+<details><summary>Example</summary>
 
-Returns the one-based index position of a substring within an expression, optionally starting from a given one-based index.
+The following example returns the one-based index of the substring `PR` in the `OriginCityName` column from the `flight-carriers` datasource starting from index 5.
+
+```sql
+SELECT 
+  "OriginCityName" AS "original_expr",
+  POSITION('PR' IN "OriginCityName" FROM 5) AS "index_found"
```

Comment:
`index_found` makes me think of a boolean. this actually returns the index. `index` is probably enough. if it is a 1-based index 0 is not in range, so understandably, not found.